### DISCUSSION TITLE
Distinct_Solutions

### DISCUSSION THREAD

**Discussion by MSAfify** [danielver02/ALPS#169] [Q&A] [answered]
Dear Prof. Kris,
Please let me know which variables I should increase to get distinct solutions using the "NHDS" dispersion solver.

Many thanks in advance!
Mahmoud

**Comment by kgklein** [comment, marked as answer]
Mahmoud- You will need to ask Daniel regarding NHDS, but in ALPS, in order to keep solutions distinct, the two variables in the scan_input namelist (see https://danielver02.github.io/ALPS/page/input.html )
ns Number of output scan values.
nres Resolution between output scan values.
control the wavevector step size, as well as swi and swf. A higher resolution in wavevector space will help keep the dispersion solutions distinct.
-Prof. Klein

**Reply by MSAfify** [reply]
Great thanks!
